index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Project 3</title>
+  <link rel="stylesheet" href="./styles/mobile-style.css">
```

Comment:
I admire your attention to details, but one CSS file is completely enough, we don't have to defragmentize them :)

Reply:
Hi Max, 

In this case, is better for me separated the CSS by feature or functionality so, I can have the code organized and this makes it easier for me to understand what I am doing. 

Thanks for your comments I will have it in mind the next time.